
<launch>
  <node name="multiMono_a" pkg="ORB_SLAM2" type="multiMono_a" output="screen" args="/home/wangkuan/workspace/ORB_SLAM2/ORB_SLAM2/Vocabulary/ORBvoc.bin   /home/wangkuan/workspace/SwarmSlam/Examples/Monocular/TUM1.yaml" >
    <remap from="/camera1/image_raw" to="/firefly1/vi_sensor/right/image_raw" />
  </node>
</launch>
